docs/configure-archive-logs.md

```diff
@@ -59,3 +61,25 @@ spec:
     archiveLocation:
       archiveLogs: true
 ```
+
+## Suggested alternatives
+
+Argo Workflows log storage facilities are quite basic.
+
+We recommend using a fully featured Kubernetes aware logging facility which will provide you with facilities for indexing, searching and managing of log storage.
+
+We also recommend adding [links](links.md) to connect from the Argo UI to your logging facility's UI. Examples are given in the [workflow-controller-configmap.yaml](workflow-controller-configmap.yaml).
+
+* Link `scope: workflow` to the logs of a Workflow
+* Link `scope: pod-logs` to the logs of a specific Pod of a Workflow
+* Parametrize the link with `${metadata.name}`, `${metadata.namespace}`, `${metadata.labels}`, and other available metadata
+
+We do not plan to reach feature parity with purpose built-logging facilities within workflows as there are more optimized tools available in the ecosystem. Some open-source products which are known to work and you could consider are
```

Comment:
```suggestion
We do not plan to reach feature parity with purpose built-logging facilities as there are more optimized tools available in the ecosystem. Some open-source tools include:
```

- [as mentioned before](https://github.com/argoproj/argo-workflows/pull/12597#discussion_r1473463034):
> - Argo is implied, since this is Argo's docs. Instead let's refer to the specific feature
>   - "logging facilities" could refer to the live logs as well, which may still have features
> - "tool" vs. "product"

- simpler and more direct, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#content-best-practices)
- add punctuation (a colon)

Reply:
The latter portion followed up in [below comment](https://github.com/argoproj/argo-workflows/pull/12597#discussion_r1486746801)